addons/editor/stringtable.xml

```diff
@@ -159,6 +159,9 @@
             <Polish>Wysadza pasażerów ze wszystkich wybranych pojazdów. Jeżeli pojazd jest w powietrzu, pasażerowie dostaną spadochron.\nJednostki zostaną wysadzone jedna po drugiej (nie wszystkie na raz).</Polish>
             <Japanese>選択された全車両から後部座席のユニットを脱出させます。もし車両が飛行中の場合はパラシュートが与えられます。\nなお脱出は一人づつ降りるようになっています (即時ではなく数秒づつ)。</Japanese>
         </Key>
+        <Key ID="STR_ZEN_Editor_UnloadViV_Description">
+            <English>Unloads all selected Vehicle-in-Vehicle cargo and carriers. A vehicle that is simultaneously both is unloaded from its carrier.</English>
```

Comment:
I'm having trouble understanding these sentences, could the phrasing be updated to be more clear?

I understand you can unload the vehicle-in-vehicle cargo with this keybind. That makes sense. But what is a carrier? I understand it as the vehicle carrying the cargo (the transporter) but how can you unload the carrier from itself? And the following sentence makes even less sense to me.

I could see interpretation issues (especially from non-natives) and translators trying to parse this text. It's just confusing :smile:

Reply:
Cargo and carrier terms come from the config:
```
class VehicleTransport {
	class Cargo {};
	class Carrier {};
};	
```

> A vehicle that is simultaneously both is unloaded from its carrier.
A vehicle can be simultaneously both, ie Box in Truck, Truck in LCU. This says that in this situation, Truck will be unloaded from LCU, not Box from Truck.

I'll try to come up with a less ambiguous phrasing.

Reply:
Maybe: "A vehicle that is both cargo and carrying cargo will only be unloaded from its carrier."

Reply:
359aad4